{
  "contest": 185,
  "handle": "sorrynotsorry",
  "risk": "2",
  "title": "CallEVM does not validate the existence of the address being called",
  "issueId": 127,
  "issueUrl": "https://github.com/code-423n4/2022-11-canto-findings/issues/127"
}